### DISCUSSION TITLE
gh.aniketsarkar.site/flask-mailing/ no more accessible ?

### DISCUSSION THREAD

**Discussion by Larsene** [marktennyson/flask-mailing#64] [Q&A] [answered]
Hi, did you see that [gh.aniketsarkar.site/flask-mailing/](https://gh.aniketsarkar.site/flask-mailing/) is no more accessible ? Where to find the docs ?

Thanks ! :-)

**Comment by marktennyson** [comment, marked as answer]
Please visit the below URL for documentation:

https://marktennyson.github.io/flask-mailing/

Thanks

**Comment by Larsene** [comment]
Many Thanks !
